CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
```

Comment:
@runfalk should I add the dev environment settings?

I usually use conda for everytthing so if you use pipenv or venv I could use a hand here, I am rusty on that side :-p

Reply:
It all should be the same with the assumption that they have a compatible python and pip installed. I don't think we need to give much guidance on conda/venv/pipenv/poetry/...

Reply:
Fair enough